src/platforms/flutter/migration.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Migration Guide
+sidebar_order: 1000
+---
+
+## Migrating from `sentry_flutter` `4.0.x` to `sentry_flutter` `5.0.0`
+
+In addition to the changes introduced in [sentry](/platforms/dart/migration/):
+
+- `SentryFlutterOptions.enableLifecycleBreadcrumbs` was replaced with `SentryFlutterOptions.enableAppLifecycleBreadcrumbs`.
+- The Web Plugin Registrant import changed from `import 'package:sentry_flutter/src/sentry_flutter_web.dart';` to `import 'package:sentry_flutter/sentry_flutter_web.dart';`
+  - This change may lead to breaking changes. In most cases, however, this change won't lead to breaking changed because the referencing file is auto-generated.
```

Comment:
```suggestion
- `SentryFlutterOptions.enableLifecycleBreadcrumbs` was replaced with `SentryFlutterOptions.enableAppLifecycleBreadcrumbs`.
- The Web Plugin Registrant import changed from `import 'package:sentry_flutter/src/sentry_flutter_web.dart';` to `import 'package:sentry_flutter/sentry_flutter_web.dart';`
- This change may lead to breaking changes. In most cases, however, this change won't lead to breaking changes because the referencing file is auto-generated.
```